### PR TITLE
Add TSC meeting notes for 04-20-2020

### PR DESCRIPTION
**Pull request by michdolan**

### CODE REVIEW

File: docs/tsc/meetings/2020-04-20.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+* PRs for review
+    - Michael: There are two Autodesk PRs waiting for review.
+    - Doug: One is related to op finalization (#986). We looked at how ops 
+      are finalized prior to evaluation, including optimization steps, 
+      converting inverse LUTs into forward LUTs, etc. There was some 
+      unnecessary cloning of ops.
+    - Patrick: We discovered cloning and slow downs in optimization and 
+      finalization and fixed them. Lots of changes, but only fixes. Some ops 
+      can have a processing cost at finalization, and we tried to avoid that. 
+      There are no high level changes; only internal processing mprovements.
```
Comment (hodoulp) [inline docs/tsc/meetings/2020-04-20.md:36]: typo 'mprovements' -> improvements

File: docs/tsc/meetings/2020-04-20.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    - Carol: I agree, and will work with Sean on this.
+    - Sean: I will make GH issues to break down tasks further.
+    
+* String locale handling
+    - Doug: PR #984 is about fixing string case handling for Turkish locale. This 
+      raises bigger case sensitivty issues. File extensions should be case 
+      insensitive for example.
+    - The proposal is to convert case as if it were english case, so that locale wouldnt 
+      matter. Two questions we need to answer. Do we want case sensitivety to depend on 
+      locale? Secondly, what areas of the library hsould use case-insensitive comparison?
```
[the remark this answers is not in the document]
Comment (michdolan) [reply docs/tsc/meetings/2020-04-20.md]: Thanks. Fixed.

File: docs/tsc/meetings/2020-04-20.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    - Patrick: Very important for integrating OCIO into DCCs, to limit 
+      performance impact.
+    - Doug: The PR feels big since it impacts all ops, but the changes are 
+      similar for each.
+    - Doug: The other PR refactors GPU usage in apps (#985). We wanted to allow 
+      CPU and GPU evaluation in apps and utilities without copying/pasting a 
+      lot of code, so this PR improves reuse between tools.
+    - Patrick: It also improves commandline tools to offer good usage examples 
+      for the library, which is important for DCC integration. It's part of 
+      ongoing imprvements in this area.
```
[the remark this answers is not in the document]
Comment (michdolan) [reply docs/tsc/meetings/2020-04-20.md]: Fixed.

File: docs/tsc/meetings/2020-04-20.md (hunk 1 above)
Comment (michdolan) [reply docs/tsc/meetings/2020-04-20.md]: Fixed.

**Pull request merged**